src/main/java/View/Web/Old/Servlets/Debugging/GENERIC_INFO.java

```diff
@@ -55,7 +55,7 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws IO
             CustomLogs.InfoLog("Exited " + SERVLET_URL + ".", true);
 
         } catch (Exception e) {
-            ServletHelper.Send404(resp);
+            ServletHelper.SendReturnCode(resp, HttpServletResponse.SC_NOT_FOUND);
```

Comment:
SendReturnCode mi pride ako vyslovene zbytocna metoda, ta povodna metoda sendError() je uz urobena tak aby to bol one-liner